jerry-core/jcontext/jcontext.h

```diff
@@ -286,6 +298,8 @@ extern jerry_hash_table_t jerry_global_hash_table;
 
 #endif /* JERRY_ENABLE_EXTERNAL_CONTEXT */
 
+
+
```

Comment:
Remove these extra lines, and as the one-line version, they occurs in several places later. Please remove those also.